DirectProgramming/DPC++/DenseLinearAlgebra/vector-add/sample.json

```diff
@@ -8,64 +9,16 @@
   "os": ["linux", "windows"],
   "builder": ["ide", "make"],
   "ciTests": {
-    "linux": [
-      {
-        "id": "cpu_buffers",
-        "steps": [
-          "make all",
-          "make run",
-          "make clean"
-        ]
-      },
-      {
-        "id": "cpu_usm",
-        "steps": [
-          "make build_usm",
-          "make run_usm",
-          "make clean"
-        ]
-      },
-      {
-        "id": "fpga_emu_buffers",
-        "steps": [
-          "make fpga_emu -f Makefile.fpga",
-          "make run_emu -f Makefile.fpga",
-          "make clean -f Makefile.fpga"
-        ]
-      },
-      {
-        "id": "fpga_report_buffers",
-        "steps": [
-          "make report -f Makefile.fpga",
-          "make clean -f Makefile.fpga"
-        ]
-      }
-    ],
-    "windows": [
-      {
-        "id": "cpu_buffers",
-        "steps": [
-          "nmake -f Makefile.win",
-          "nmake -f Makefile.win run",
-          "nmake -f Makefile.win clean"
-        ]
-      },
-      {
-        "id": "cpu_usm",
-        "steps": [
-          "nmake -f Makefile.win build_usm",
-          "nmake -f Makefile.win run_usm",
-          "nmake -f Makefile.win clean"
-        ]
-      },
-      {
-        "id": "fpga_emu_buffers",
-        "steps": [
-          "nmake -f Makefile.win.fpga",
-          "nmake -f Makefile.win.fpga run",
-          "nmake -f Makefile.win.fpga clean"
-        ]
-      }
-    ]
-  }
+    "linux":[
+      {"id": "cpu_buffers", "build": ["make all"],"run": ["make run"],"clean": ["make clean"]},
+      {"id": "cpu_usm","build": ["make build_usm"],"run": ["make run_usm"],"clean": ["make clean"]},
+      {"id": "fpga_emu_buffers","build": ["make clean -f Makefile.fpga","make fpga_emu -f Makefile.fpga"],"run": ["make run_emu -f Makefile.fpga"]},
+      {"id": "fpga_report_buffers","build": ["make report -f Makefile.fpga"],"clean": ["make clean -f Makefile.fpga"]}
+      ],
+	"windows":[
+	  {"id": "cpu_buffers","build": ["nmake -f Makefile.win"],"run": ["nmake -f Makefile.win run"],"clean": ["nmake -f Makefile.win clean"]},
+      {"id": "cpu_usm","build": ["nmake -f Makefile.win build_usm"],"run": ["nmake -f Makefile.win run_usm"],"clean": ["nmake -f Makefile.win clean"]},
+      {"id": "fpga_emu_buffers","build": ["nmake -f Makefile.win.fpga clean","nmake -f Makefile.win.fpga"],"run": ["nmake -f Makefile.win.fpga run"]}
+	  ]
+    }
```

Comment:
The existing configurations are the correct ones in terms of syntax. Please, review them and make required updated if needed.

Reply:
I'll make the change and resubmit.

Reply:
@mkitez Jennifer is on vacation, can you help me understand what the required updates are?

Reply:
@sravanikonda  The json field's value in this PR is outdated. It is not using the new format specified here : https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object